### ISSUE TITLE
Why did Mac output "zsh: command not found: mapf-visualizer"?

### ISSUE THREAD

**Issue by fengrn23** [Kei18/mapf-visualizer#15]
Hello, I had downloaded mapf-visualizer on my Mac successfully, but when I input the command "mapf-visualizer assets/random-32-32-20.map assets/demo_random-32-32-20.txt", the terminal output "zsh: command not found: mapf-visualizer". Why did it happen? What should I do? Thank you!

<img width="567" alt="截屏2024-06-04 15 08 27" src="https://github.com/Kei18/mapf-visualizer/assets/159398915/bd44d4f1-c28e-4091-8441-90f99119450c">

**Comment by Kei18** [comment]
The path does not seem to be registered correctly. You can try .zshrc instead of .bashrc:

```sh
echo "export PATH=$(pwd)/bin/mapf-visualizer.app/Contents/MacOS:\$PATH" >> ~/.zshrc
```

Or you can directly call `[...]/bin/mapf-visualizer.app/Contents/MacOS/mapf-visualizer`.

Cheers,
Keisuke

**Comment by mittal-ishaan** [comment]
@Kei18 
I can create a PR for updating the readme here for this case.

**Comment by Kei18** [comment]
resolved by #16
